cmd/server/main.go

```diff
@@ -189,19 +188,20 @@ func realMain(ctx context.Context) error {
 		sub.Handle("", homeController.HandleHome()).Methods("GET")
 
 		// API for creating new verification codes. Called via AJAX.
+		issueapiController := issueapi.New(ctx, config, db, h)
 		sub.Handle("/issue", issueapiController.HandleIssue()).Methods("POST")
 	}
+
 	{
-		sub := r.PathPrefix("/cstatus").Subrouter()
+		sub := r.PathPrefix("/codestatus").Subrouter()
```

Comment:
I'm thinking this should be `/codes/status`. We can always change it later, but it'd be good to do in now and not have to worry about redirects.